templates/registration/register.html.twig

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
could you merge this part with login page in a base template and extends from it here?